README.md

```diff
@@ -64,12 +64,10 @@ ftpDeploy.on('uploaded', function(data) {
 To continue uploading files even if a file upload fails: 
```

Comment:
Maybe better to change to "To continue uploading files even if a file upload fails, add these before `ftpDeploy.deploy`:".